### ISSUE TITLE
paper-dropdown-menu inside a paper-dialog 

### ISSUE THREAD

**Issue by alecf** [PolymerElements/iron-fit-behavior#21]
I hit this as well when using a paper-dropdown-menu deep inside a paper-dialog 

If you start with the dialog closed, and then call dialog.open(), the iron-resize event seems to cause a short cascade of sizing inside iron-overlay-behavior, where the dialog briefly sizes normally and then settles at a tiny dialog. If you manually call fit() the dialog goes to the right size.

But then, when you'd click on the paper-dropdown-menu, this triggers an iron-resize, (from the iron-dropdown) and that in turn would cause the iron-overlay-behavior to try to reposition/recenter the dialog, which gives you the tiny dialog again.

The issue I seemed to hit is that the iron-overlay-behavior was calling this.refit() to do the resizing. this.resetFit() seems to mess around with positions on the element itself (i.e. not just clear the cached state) and those positions are used later to calculate the position in fit()

basically it seems like refit() is not leaving the element in a safe state. If I change the refit() call in iron-overlay-behavior to just be fit(), then the dialog sizes correctly when the paper-dropdown-menu is clicked.

**Comment by barakhub** [comment]
@alecf - I encountered some issues with paper-dialog not opening at the right size as well, and they're unrelated to having a dropdown in them.

Can you try adding `position: fixed` to your `<paper-dialog>` and see if this fixes the issue?

**Comment by barakhub** [comment]
Note: if adding `position: fixed` solves your problem, this issue technically a dup of issue #20 (and actually many others).

**Comment by alecf** [comment]
Finally getting around to this - yes, position: fixed fixes this problem. Thanks!
